packages/transformers/babel/src/config.js

```diff
@@ -29,6 +29,20 @@ export async function load(
     return buildDefaultBabelConfig(config);
   }
 
+  // If we are in a monorepo, also find .babelrc configs in the sub packages.
+  let babelrcRoots = [options.projectRoot];
+  let packageJSONPath = await resolveConfig(
+    options.inputFS,
+    config.searchPath,
+    ['package.json'],
+  );
+  if (packageJSONPath) {
+    let packageRoot = path.dirname(packageJSONPath);
+    if (packageRoot && packageRoot !== options.projectRoot) {
+      babelrcRoots.push(packageRoot);
```

Comment:
We probably also need to track the package.json paths we searched as part of the config so it will be invalidated in the cache properly. Maybe a glob? cc. @padmaia

Reply:
Oh yeah, I guess that never got set. Easy to add though, it's just a call to `config.setWatchGlob()`.

Reply:
I'm not sure I fully understand how the cache invalidation works - could you elaborate on what you expect to happen (but isn't already), and how I might be able to test it?

In case it's relevant, one thing to note is that the sub-project babel config might be specified in a `.babelrc`/`babelrc.json` file in the sub-package directory, or as part of the `babel` property of the `package.json` file. Presumably, we'd want to make sure that the bundle will get regenerated if any of these change (and do whatever is necessary to keep the cache clean). Given this, are there any other tweaks you think might be necessary?